recipes/default.rb

```diff
@@ -24,10 +24,21 @@
 # ::Chef::Recipe.send(:include, Chef::Mixin::PowershellOut)
 ::Chef::Resource::RubyBlock.send(:include, Chef::Mixin::PowershellOut)
 
-powershell_script 'install chocolatey' do
-  code "iex ((new-object net.webclient).DownloadString('#{node['chocolatey']['Uri']}'))"
-  convert_boolean_return true
-  not_if { ChocolateyHelpers.chocolatey_installed? }
+if File.exist?('C:\windows\system32\cmd.exe')
+  arch = :x86_64
+  cmd = 'C:\windows\system32\cmd.exe'
```

Comment:
Yeah, it feels like that would be additional complication that would only add possibly unintended consequences...    This now reads like the logic in my brain which is `if we're doing SysWOW64 stuff then special case it, otherwise just let it run normally....`